packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -1642,8 +1642,7 @@ void main() {
 
       // But don't trigger the toolbar.
       expect(find.byType(CupertinoButton), findsNothing);
-    },
-  );
+  }, variant: const TargetPlatformVariant(<TargetPlatform>{ TargetPlatform.iOS,  TargetPlatform.macOS }));
```

Comment:
All of these tests that I added specific platforms to were testing Apple-specific behavior on all platforms.  Before I moved that platform switch out of TextField and into the shared TextSelectionGestureDetectorBuilder class, CupertinoTextField was incorrectly forcing all platforms to have single taps that behaved like iOS/Mac.  Now taps will behave however the platform should behave regardless of what type of field is involved.